AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/EFragmentHolder.java

```diff
@@ -134,7 +136,7 @@ private void setFindViewById() {
 	}
 
 	private void setFragmentBuilder() throws JClassAlreadyExistsException {
-		fragmentBuilderClass = generatedClass._class(PUBLIC | STATIC, "FragmentBuilder_");
+		fragmentBuilderClass = generatedClass._class(PUBLIC | STATIC, "FragmentBuilder" + classSuffix());
```

Comment:
I do not agree. Generally we use this impl classes when there is an interface for a specific work, and then we provide an implementation for that work. Here, we do not provide implementation for the work, but we generate always the interface actually, too.